### DISCUSSION TITLE
BigQuery integration and features

### DISCUSSION THREAD

**Discussion by Rhythmfish** [dataform-co/dataform#1395] [General]
I've been following the GCP BigQuery integration very closely as I plan on integrating it into my team's workflow. Unfortunately, it's hard to find any information about the ongoing migration. I managed to hit this awkward window where you can't get the standalone Dataform product and it's only available in a Preview build in BigQuery.
It feels like functionality is limited in BQ in its current state and I'm curious to find out how each function that Dataform used to offer will translate. What's the feature map look like and what are the goals for when Dataform goes to the GA build in GCP?  
An example of translation is how scheduling jobs now appears to require a completely separate cloud product: Workflows in combination with Cloud Scheduler or Cloud Composer.  Are any other core functionalities going to be stripped out and handled by other google products?

**Comment by BenBirt** [comment]
Thanks for the questions!

Suffice it to say that GCP Dataform is certainly not feature-complete, yet. We are building out features as I type!

For details of upcoming plans, it is best to get in touch with us via your Technical Account Manager / Customer Engineer, or directly to our Product Manager ghh@google.com.
